Library/Homebrew/tap.rb

```diff
@@ -31,7 +31,7 @@ def self.fetch(*args)
     repo = repo.strip_prefix "homebrew-"
 
     if user == "Homebrew" && repo == "homebrew" ||
-        user == "Linuxbrew" && repo == "linuxbrew"
+        repo == "linuxbrew"
```

Comment:
Sorry, the regex should be `%r{^([\w-]+)/homebrew-([\w-]+)$}`. I wrote it wrong in the first time.